chp00_introduction/NOC_I_01_RandomWalkTraditional/sketch.js

```diff
@@ -15,16 +15,18 @@ function draw() {
   walker.render();
 }
 
-function Walker() {
-  this.x = width/2;
-  this.y = height/2;
+class Walker {
+  constructor(){
+    this.x = width/2;
+    this.y = height/2;
+  };
```

Comment:
Are these semi-colons necessary? I think maybe I prefer to leave them out?

Reply:
There were a number of semi-colons in the original code that I left in because I wasn't sure if they were necessary. I think it's cleaner w/o them.